dom/attr.ts

```diff
@@ -45,12 +42,8 @@ export function setAttrs (element, attributes)
  * Returns the attribute value for the given html node
  *
  * @deprecated use `element.getAttribute()` directly instead. Will be removed in v3.0
- *
- * @param {Element} element
- * @param {string} attribute
- * @returns {string | null}
  */
-export function getAttr (element, attribute)
+export function getAttr (element : Element, attribute : string) : null|string
```

Comment:
Can you send two additional PRs?

One where you remove this function in master.
And another one where you deprecate it in v2.